src/app/components/tabs/tabs-demo.component.html

```diff
@@ -11,3 +11,15 @@
     This tab cannot be closed.
   </sky-tab>
 </sky-tabset>
+
+<h3>Tabs example with counter</h3>
```

Comment:
Also, could we have a demo with a count and a close x? That way it shows that you can do this.